civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -10,33 +10,88 @@ use Drupal\civihr_employee_portal\Helpers\HelperClass;
  */
 function civihr_employee_portal_css_alter(&$css) {
 
-    // Remove defaults system messages css file.
-    unset($css[drupal_get_path('module', 'system') . '/system.messages.css']);
+  // Remove defaults system messages css file.
+  unset($css[drupal_get_path('module', 'system') . '/system.messages.css']);
+
+  // Removing unwanted css files from civicrm pages.
+  if (civicrm_invoke()) {
+    // Multi-dimensional array containing module name and file path.
+    $removeFiles = [
+      ['module_name' => 'ctools', 'file_name' => '/css/ctools.css'],
+      ['module_name' => 'date_api', 'file_name' => '/date.css'],
+      [
+        'module_name' => 'date_popup',
+        'file_name' => '/themes/datepicker.1.7.css',
+      ],
+      ['module_name' => 'panels', 'file_name' => '/css/panels.css'],
+      ['module_name' => 'radix_layouts', 'file_name' => '/radix_layouts.css'],
+      ['module_name' => 'views_tooltip', 'file_name' => '/views_tooltip.css'],
+      ['module_name' => 'tipsy', 'file_name' => '/stylesheets/tipsy.css'],
+    ];
+    // Removes above files.
+    _remove_alter_files_from_civicrm($css, $removeFiles);
+  }
 }
 
 /**
  * Implements hook_js_alter()
  */
 function civihr_employee_portal_js_alter(&$javascript) {
 
-    // Unset jcarousel on panel admin pages as it's causes infinite loops
-    // "admin/structure/pages"
-    if (strpos(current_path(), 'admin/structure/pages') !== false) {
-        foreach ($javascript['settings']['data'] as $key => $value) {
-            if (isset($value['jcarousel'])) {
-
-                unset($javascript['settings']['data'][$key]);
-            }
-        }
-
+  // Unset jcarousel on panel admin pages as it's causes infinite loops
+  // "admin/structure/pages"
+  if (strpos(current_path(), 'admin/structure/pages') !== FALSE) {
+    foreach ($javascript['settings']['data'] as $key => $value) {
+      if (isset($value['jcarousel'])) {
+        unset($javascript['settings']['data'][$key]);
+      }
     }
+  }
 
-    global $user;
-    $uf = get_civihr_uf_match_data($user->uid);
-    drupal_add_js(array('currentDrupalUserId' => $user->uid), 'setting');
-    drupal_add_js(array('currentCiviCRMUserId' => $uf['contact_id']), 'setting');
+  global $user;
+  $uf = get_civihr_uf_match_data($user->uid);
```

Comment:
I think the code would be more readable if you put both 
```php
global $user;
$uf = get_civihr_uf_match_data($user->uid);
```
at the top of the function